lls/nd.c

```diff
@@ -529,6 +542,10 @@ process_nd_neigh_solicitation(struct lls_config *lls_conf, struct rte_mbuf *buf,
 
 		icmpv6_hdr->cksum = rte_ipv6_icmpv6_cksum(ipv6_hdr, icmpv6_hdr);
 
+		ret = adjust_shift_l2_hdr(iface, buf, eth_hdr);
+		if (ret < 0)
+			return -1;
```

Comment:
As in my previous comments, a comment explaining why we don't need `adjust_shift_l2_hdr()` suffices.